content/applications/general/multi_company.rst

```diff
@@ -0,0 +1,114 @@
+=============
+Multi-company
+=============
+
+.. |mcd| replace:: multi-company database
+
+In Odoo, multiple companies can exist within a single database. This allows for some data to be
+shared among companies, while still maintaining some level of separation between entities.
+
+Before deciding to use the multi-company feature, there are several factors to consider.
+
+.. important::
+   Multi-company is **only** available in One-app free databases, or with
+   `Custom <https://www.odoo.com/pricing-plan>`_ plans.
+
+Accessing multiple companies
+============================
+
+The list of companies an employee :ref:`has access <general/employee-access>` to in a |mcd| can be
+found at the top-right of the main Odoo menu bar, where the active company is listed. Click on the
+company name to reveal a list of all allowed companies. To switch to a different company, click on
+the company name. To enable multiple companies at once, tick the checkbox next to each applicable
+company name.
+
+.. figure:: multi_company/company-access.png
+   :align: center
+   :alt: An example of the list of companies a user has access to when logged into a database.
+
+   An example of a user with access to multiple companies. The current company is My Company (San
+   Francisco), while My Company (Chicago) is also active.
+
+.. note::
+   The database may refresh after each checkbox is ticked.
+
+.. _general/active-companies:
+
+Multiple active companies
+-------------------------
+
+If more than one company is active at a time, one company is highlighted in purple, and is listed on
+the menu bar. This is the considered the *current* company.
+
+When creating a new record, the *current* company is added to the record in the :guilabel:`Company`
```

Comment:
```suggestion
When creating a new record, the *current* company is added to the record in the *Company*
```
suggesting the use of italics here, because there is no *specific* field with this name that is currently being discussed/shown on the screen/UI